recipes/sphinx-autobuild/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: sphinx-autobuild
+  version: "0.6.0"
+
+source:
+  fn: sphinx-autobuild-0.6.0.tar.gz
+  url: https://pypi.python.org/packages/85/cf/25b65781e6d2a4a89a431260daf1e0d53a81c52d27c98245481d46f3df2a/sphinx-autobuild-0.6.0.tar.gz
+  md5: 7331bb8e14b537b871ab8e6e09ff4713
+
+build:
+  # noarch_python: True
+  preserve_egg_dir: True
+  entry_points:
+    - sphinx-autobuild=sphinx_autobuild:main
+  number: 0
+
+requirements:
+  build:
```

Comment:
If you use the `script` command inside the yaml file mentioned in another comment, I believe you will only need `python` and `setuptools` as `build` requirements.